src/catalog/src/config.rs

```diff
@@ -118,14 +118,12 @@ impl ClusterReplicaSizeMap {
             CatalogError::Catalog(SqlCatalogError::UnknownClusterReplicaSize(name.into()))
         })
     }
-}
 
-impl Default for ClusterReplicaSizeMap {
-    // Used for testing and local purposes. This default value should not be used in production.
-    //
-    // Credits per hour are calculated as being equal to scale. This is not necessarily how the
-    // value is computed in production.
-    fn default() -> Self {
+    /// Used for testing and local purposes. This default value should not be used in production.
+    ///
+    /// Credits per hour are calculated as being equal to scale. This is not necessarily how the
+    /// value is computed in production.
+    pub fn for_tests() -> Self {
```

Comment:
This change is load-bearing because we remove the default implementation. Code that depends on a value need to use `for_tests`, which should make it sufficiently clear that this value isn't to be used in many situations.